### PR TITLE
Rough outline of FAQ Page with Collapsible Questions

### PR DESCRIPTION
**Pull request by Hackin7**
image

Questions can be input in via JSON (may need to figure out a better way to store the JSON?)

### CODE REVIEW

**Comment by wei2912** [comment]
Would be good if we can switch to using https://mui.com/material-ui/react-accordion/ for the dropdowns.

**Comment by Hackin7** [comment]
image

To add relevant questions. Not too sure where to put the questions (or if it even makes sense to separate it from the page tsx)

File: pages/faq.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        <h1>Frequently Asked Questions</h1>
+        <div>
+          {questions.map(({ question, answer }, index) => (
+            <Accordion>
+              <AccordionSummary
+                expandIcon={<ExpandMoreIcon />}
+                aria-controls="panel1a-content"
+                id="panel1a-header"
+              >
+                <Typography>{question}</Typography>
```
Comment (wei2912) [inline pages/faq.tsx:51]: Could we bold the question?

**Review by wei2912 — CHANGES_REQUESTED** [review]
LGTM, but would be good if we could import the JSON object from another file, rather than hardcode into `pages/faq.tsx`.

**Review by wei2912 — APPROVED** [review]
Ignore "Request Changes" in the previous comments.

**Pull request merged**